src/App.jsx

```diff
@@ -14,45 +17,113 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button
-        type="button"
-        className="button is-info is-light"
-      >
-        Sort alphabetically
-      </button>
-
-      <button
-        type="button"
-        className="button is-success is-light"
-      >
-        Sort by length
-      </button>
-
-      <button
-        type="button"
-        className="button is-warning is-light"
-      >
-        Reverse
-      </button>
-
-      <button
-        type="button"
-        className="button is-danger is-light"
-      >
-        Reset
-      </button>
-    </div>
+const SORT_FIELD_ALPH = 'alph';
+const SORT_FIELD_LENGTH = 'length';
+
+function getPreparedGoods(
+  goods,
+  sortField,
+  reverse,
+) {
+  let preparedGoods = [...goods];
+
+  if (sortField) {
+    switch (sortField) {
+      case SORT_FIELD_ALPH:
+        preparedGoods.sort((good1, good2) => good1.localeCompare(good2));
+        break;
+      case SORT_FIELD_LENGTH:
+        preparedGoods.sort((good1, good2) => good1.length - good2.length);
+        break;
+      default:
+        preparedGoods = [...preparedGoods];
```

Comment:
```suggestion
        break;
```
you've already made a copy above - no need to do it again

Reply:
agree, please do it